dotnet/src/SemanticKernel/Memory/VolatileMemoryStore.cs

```diff
@@ -3,7 +3,6 @@
 using System.Collections.Generic;
```

Comment:
Hey @tawalke please confirm these deletions are expected.

Reply:
There should have been no changes to VolatileMemoryStore.cs. This must be a bad merge issue.